Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -125,6 +126,8 @@ class TextureCacheBase
   virtual void DeleteShaders() = 0;
 
   TCacheEntry* Load(const u32 stage);
+  static void InvalidateAllBindPoints() { invalidated_binds = 0xff; }
```

Comment:
If you made it a bitset and there was a constant saying how many binds there were (total_bind_points) this would be something like

```
invalidate_binds = std::bitset<total_bind_points>().set();
```